components/typography/typography-styles.js

```diff
@@ -1,85 +1,109 @@
 import { css } from 'lit-element/lit-element.js';
 
-export const typographyStyles = css`
-	.d2l-body-standard-text {
+export const bodyStandardStyles = css`
+	.d2l-body-standard {
 		font-size: 0.95rem;
 		font-weight: 400;
 		line-height: 1.4rem;
 	}
-	.d2l-body-compact-text {
+	@media (max-width: 615px) {
+		.d2l-body-standard {
+			font-size: 0.8rem;
+			line-height: 1.2rem;
+		}
+	}
+`;
```

Comment:
Nit: I might drop `typography` from the filename and `Styles` from the end of all the export names? Seems redundant if you're doing:

    import {bodyStandard, heading1} from 'd2l-core/components/typography/styles.js';